projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -9,10 +9,13 @@
     "cy:open:b2b": "cypress open --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers",
     "cy:run": "cypress run",
     "cy:run:b2b": "cypress run --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers",
+    "cy:run:core": "cypress run --spec \"cypress/integration/**/*.core-e2e-spec.ts\"",
     "cy:run:ci": "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2c,all,parallel\" --parallel --group B2C --ci-build-id $TRAVIS_BUILD_ID --spec \"cypress/integration/!(vendor|b2b|ssr)/**/*.e2e-spec.ts\"",
-    "cy:run:ci:ssr": "cypress run --config-file $CYPRESS_CONFIG --config baseUrl=http://localhost:4000 --record --key $CYPRESS_KEY --tag \"2011,ssr,all,parallel\" --parallel --group SSR --ci-build-id $TRAVIS_BUILD_ID --spec \"cypress/integration/ssr/pages.e2e-spec.ts\"",
-    "cy:run:ci:b2b": "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2b,all\" --group B2B --ci-build-id $TRAVIS_BUILD_ID --spec \"cypress/integration/b2b/**/*.e2e-spec.ts\"",
-    "cy:run:ci:cds": "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2c,all-cds\" --group CDS --spec \"cypress/integration/vendor/cds/**/*.e2e-spec.ts\"",
+    "cy:run:ci:core" : "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2c,all,parallel\" --parallel --group B2C --ci-build-id $TRAVIS_BUILD_ID --spec \"cypress/integration/!(vendor|b2b|ssr)/**/*.core-e2e-spec.ts\"",
```

Comment:
remove the `all` tag and add a `core` tag. It applies to all `core` targets

Reply:
done